configs/dev-config.yaml

```diff
@@ -21,6 +21,21 @@ asset:
 
 inspector_url: http://localhost:3333/dist/aframe-inspector.js
 
+default_data:
+  transitions:
+    [
+      {
+        "data":
+          [
+            {
+              "text": "INTRODUCTION: text goes here.",
+              "imageSrc": "image goes here",
+            },
+          ],
+      },
```

Comment:
Similarly, let's add the `currPosition` to the default. Basically, if we were to use this default, we want to ensure nothing breaks.